examples/angular/src/pages/ui/components/authenticator/custom-slots/custom-slots.component.html

```diff
@@ -21,7 +21,7 @@
   <ng-template amplifySlot="sign-in-footer" let-authenticator="authenticator">
     <div style="text-align: center">
       <button
-        (click)="authenticator.toResetPassword()"
+        (click)="auth.toResetPassword()"
```

Comment:
@ErikCH I think I'm going to tweak `get auth()` to be `get authenticator()` so there's not confusion between this & the Amplify `Auth` category methods.

Reply:
ok!